CHANGELOG.md

```diff
@@ -32,7 +34,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     - Updated PDR and Granule writes to check the step function
     workflow_start_time against the createdAt field for each record to
     ensure old records do not overwrite newer ones for legacy Dynamo and
-    Postgres writes
+    PostgreSQL writes
```

Comment:
Is this a convention we're adopting across the board?    I think I'd used Postgres, generally in the documentation.

Reply:
Yeah, in the change log it's all (or almost all) `PostgreSQL`. Here I'm just conforming to what's there. Happy to change it but it should probably be consistent so I'd change all references in the change log to `Postgres`. 

PostgreSQL is a bit more formal I suppose but I don't feel strongly either way.

Reply:
Ugh, ditto.